### DISCUSSION TITLE
Aiuto

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#658] [Richieste di aiuto]
Non riesco a fare lo scrutinio risulta Error 500

**Comment by trinko** [comment]
Devi fornire tutti i dettagli...
Vedi [Come fare una segnalazione di errore](https://github.com/iisgiua/giuaschool/discussions/576)

**Reply by luigirotondi** [reply]
[2024-03-04_07-27.log (1).txt](https://github.com/iisgiua/giuaschool/files/14478383/2024-03-04_07-27.log.1.txt)
